doc/nrf/images/zigbee_signal_handler_01_production_config.uml

```diff
@@ -0,0 +1,9 @@
+@startuml
+
+:ZB_ZDO_SIGNAL_PRODUCTION_CONFIG_READY;
+
+:NRF_LOG_INFO(load_status);
```

Comment:
`NRF_LOG_INFO` unavailable in NCS.
According to our newest guidelines - should it be replaced with a simple comment?